src/pudl/output/ferc1.py

```diff
@@ -1133,21 +1202,18 @@ def calculations(self: Self):
         return calc_explode
 
     @property
-    def metadata(self):
+    def metadata(self: Self) -> pd.DataFrame:
```

Comment:
@cmgosnell and I talked a couple of weeks ago about how re-organizing the relationships between `Exploder`, `MetadataExploder` and `CalculationForest`, and also about factoring the metadata transformations out of the data transformation classes since it's a whole separate parallel system now, but we put it off and I don't remember right now how we were thinking about organizing it.  It does seem like we could just turn these into `Exploder.exploded_meta` and `Exploder.exploded_calcs` properties.

Reply:
I went ahead and consolidated these classes.